### DISCUSSION TITLE
Plotting Geo Data with Plotnine

### DISCUSSION THREAD

**Discussion by CarlosVecina** [has2k1/plotnine#856] [2024 Plotnine Contest]
### Authors

Carlos Vecina Tebar

### Links

https://github.com/CarlosVecina/random_vizs/tree/master/plotnine_geo_data

<img width="634" alt="Captura de pantalla 2024-07-26 a las 14 39 15" src="https://github.com/user-attachments/assets/3dad8497-3850-443d-9574-3fd0521a29f1">

### Full description

Here is an example of how to use geo data with Plotnine.

_Context_
Imagine you are a emergency response company. You have several stations spread across the country to attend to different alerts.

_Notebook explanation_
We are loading the Spanish province polygons in Shap format, generating some information, and then plotting it with the awesome Plotnine module.

<br>

I still enjoy and find it hard to stop using this plot grammar!✨🙌🏻

### Code repository

https://github.com/CarlosVecina/random_vizs/tree/master/plotnine_geo_data

**Comment by has2k1** [comment]
![download (4)](https://github.com/user-attachments/assets/a51a2f66-e346-4da6-bd22-aeb0a81590d1)

**Comment by EconomiCurtis** [comment]
Ahoy @CarlosVecina , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Comment by has2k1** [comment]
@CarlosVecina, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
